UDV-Core/src/Utils/BaseDemo/js/BaseDemo.js

```diff
@@ -408,49 +388,101 @@ export class BaseDemo {
             source: wmsElevationSource,
         });
         this.view.addLayer(wmsElevationLayer);
+    }
 
-        // ********* ADD 3D BUILDING LAYER (3D Tiles)
-        // This building layer represents Lyon in 2015 and is served from
-        // grand lyon alpha server
+    /**
+     * Adds a 3D Tiles layer to the iTowns 3D view.
+     * @param {string} layer The name of the type of object to add to the view. This name should
+     * be one of the properties of the 3DTilesLayer object (in .json config file).
+     */
```

Comment:
Maybe rename layer to layerConfig for more clarity and describe its structure and add a link to an example (i.e. to the generalDemoConfig.json file).